Dockerfile.template

```diff
@@ -124,7 +125,20 @@ RUN set -ex; \
 	\
 	apt-mark auto '.*' > /dev/null; \
 	[ -z "$savedAptMark" ] || apt-mark manual $savedAptMark; \
-	apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false
+	apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false; \
+	echo {{
+		{
+			name: "gcc",
+			version: .version,
+			params: {
+				os_name: "debian",
+				os_version: .debian.version
+			},
+			licenses: [
+				"Apache-2.0"
+			]
+		} | sbom | tostring | @sh
+	}} > /usr/local/gcc.spdx.json;
```

Comment:
I've obviously missed a lot, so apologies, but why is this just sitting raw in `/usr/local` and not a subdirectory? :grimacing:

Reply:
Typically we install it under the same directory structure where the application is installed. When the installation is under `/opt`, we have put it in the app subdirectory, e.g., https://github.com/docker-library/rabbitmq/blob/b6b4cd71df7d15918afbd4e7ab021266bb2310b7/Dockerfile-ubuntu.template#L234-L264 . When an app is installed under `/usr/local`, the SPDX file should probably be under `/usr/local/lib`, since despite it being architecture independent, which would imply `share`, it points to binary files. Perhaps something like `/usr/local/lib/sbom`?